spec/book_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'spec_helper'
+
+describe Book do
+  before :each do
+    @book = Book.new('Title', 'Author')
+  end
+
+  it 'should return a Book object' do
+    expect(@book).to be_an_instance_of Book
+  end
+
+  describe '#title' do
+    it 'should return a book title' do
+      expect(@book.title).to eql 'Title'
+    end
+  end
+
+  describe '#author' do
+    it 'should return a book author' do
+      expect(@book.author).to eql 'Author'
+    end
+  end
+end
+
+describe Book do
+  before :each do
+    @book = Book.new('Title', 'Author')
+  end
+
+  it 'should return a Book object' do
+    expect(@book).to be_an_instance_of Book
+  end
+
+  describe '#title' do
+    it 'should return a book title' do
+      expect(@book.title).to eql 'Title'
+    end
+  end
+
+  describe '#author' do
+    it 'should return a book author' do
+      expect(@book.author).to eql 'Author'
+    end
+  end
+end
```

Comment:
- [ ] Kindly add tests for the `add_rental` and `to_json` methods in the Book class. We are required to write tests for methods in the classes, not just the initializers